tests/moduletool/test_install.py

```diff
@@ -488,6 +490,8 @@ def test_project_install(
     # autostd=True reports std as an import for any module, thus requiring it to be v2 because v2 can not depend on v1
     module.Project.get().autostd = False
     ProjectTool().execute("install", [])
+    # Make sure update works in all cases where install passed
+    ProjectTool().execute("update", [])
```

Comment:
By always doing the update, don't we lose the assertions on the install?

Reply:
I'll move it